sui_programmability/examples/nfts/sources/random_word.move

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/// A NFT shop that creates NFTs with random words and can be minted without going through consensus.
+/// It is an example of a secure auto generated NFTs factory, e.g. for selling game items.
+///
+/// The NFT creator defines the possible words and their weights.
+/// Anyone can purchase a word from the shop, chosen randomly for the user according to the distribution defined by
+/// the weights.
+///
+module nfts::random_word {
+    use sui::object::{Self, UID, ID};
+    use sui::tx_context::{Self, TxContext};
```

Comment:
I would suggest importing:
```
use sui::tx_context::{sender, TxContext};
```

Since the only function used below is `sender(ctx)`.